core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -85,15 +85,20 @@
     private boolean saveKeyValues;
     private TableId tableId;
     private ReadConsistency readConsistency = ReadConsistency.IMMEDIATE;
+    private AccumuloClient _client;
+
+    Builder(AccumuloClient client) {
+      this._client = client;
+    }
 
     @Override
-    public TabletsMetadata build(AccumuloClient client) {
-      Preconditions.checkState(level == null ^ table == null);
+    public TabletsMetadata build() {
+      Preconditions.checkState((level == null) != (table == null));
```

Comment:
This precondition should have an accompanying description.

Reply:
I'm not sure what this is guarding against... only one can be null and only one set? I feel like this would be less confusing as two separate checks.

Reply:
For description: "Only the table or the level can be set" ? I am not sure that makes sense in the context of the builder.

Reply:
Maybe "scanTable() cannot be used in conjunction with forLevel() or forTable()". I didn't look too deep into why these can't be used together, but that seems to be the restriction that is being enforced.

Reply:
Also looks like `forTablet()` as well. I will include it in your suggestion.

Reply:
I also just attempted to write a test for this error and it would be difficult since the builder interfaces protect against this condition.